docs/scos/dev/glue-api-guides/202200.0/managing-wishlists/managing-wishlist-items.md

```diff
@@ -0,0 +1,258 @@
+
+### Request
+
+| HEADER KEY | HEADER VALUE | REQUIRED | DESCRIPTION |
+| --- | --- | --- | --- |
+| Authorization | string | &check; | Alphanumeric string that authorizes the customer to send requests to protected resources. Get it by [authenticating as a customer](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-customers/authenticating-as-a-customer.html).  |
+
+<details><summary>Request sample</summary>
+
+`POST https://glue.mysprykershop.com/wishlists/09264b7f-1894-58ed-81f4-d52d683e910a/wishlist-items`
+
+```json
+{
+		"data": {
+			"type": "wishlist-items",
+			"attributes": {
+				"sku": "064_18404924"
+			}
+		}
+	}
+```
+
+</details>
+
+<details><summary>Request sample: add a configurable product to a wishlist</summary>
+
+`POST https://glue.mysprykershop.com/wishlists/09264b7f-1894-58ed-81f4-d52d683e910a/wishlist-items`
+
+```json
+{
+    "data": {
+        "type": "wishlist-items",
+        "attributes": {
+            "sku": "093_24495843",
+            "productConfigurationInstance": {
+                "displayData": "{\"Preferred time of the day\": \"Afternoon\", \"Date\": \"9.12.2021\"}",
+                "configuration": "{\"time_of_day\": \"2\"}",
+                "configuratorKey": "DATE_TIME_CONFIGURATOR",
+                "isComplete": true,
+                "quantity": 3,
+                "availableQuantity": 4,
+                "prices": [
+                    {
+                        "priceTypeName": "DEFAULT",
+                        "netAmount": 23434,
+                        "grossAmount": 42502,
+                        "currency": {
+                            "code": "EUR",
+                            "name": "Euro",
+                            "symbol": "€"
+                        },
+                        "volumePrices": [
+                            {
+                                "netAmount": 150,
+                                "grossAmount": 165,
+                                "quantity": 5
+                            },
+                            {
+                                "netAmount": 145,
+                                "grossAmount": 158,
+                                "quantity": 10
+                            },
+                            {
+                                "netAmount": 140,
+                                "grossAmount": 152,
+                                "quantity": 20
+                            }
+                        ]
+                    }
+                ]
+            }
+        }
+    }
+}
+```
+
+</details>
+
+<a name="request-attributes-description"></a>
```

Comment:
Is it normal to have a link without an HREF tag?

Reply:
this is an anchor to which I'm referring to from another place in the document